pkg/operator/ceph/file/mds/mds.go

```diff
@@ -219,11 +314,10 @@ func (c *Cluster) scaleDownDeployments(replicas int32, desiredDeployments map[st
 	for _, d := range deps.Items {
 		if _, ok := desiredDeployments[d.GetName()]; !ok {
 			// if deployment name is NOT in improvised set, delete it
-			logger.Infof("Deleting extraneous mds deployment %s", d.GetName())
+			logger.Infof("Deleting or stopping mds deployment %s", d.GetName())
```

Comment:
This path already knows if it is a delete or not by the `delete` variable. I have done something like this before.
```go
deletingOrStopping := "deleting"
if !delete {
    deletingOrStopping = "stopping"
}
logger.Infof("%s MDS deployment %q", deletingOrStopping, d.GetName())
```